docs/platform-engineers/advanced-install.mdx

```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 
 ### 1. Prerequisites:
 
-- Kubernetes cluster >= v1.19 && <= v1.23
+- Kubernetes cluster >= v1.19 && <= v1.24
```

Comment:
does 1.5.3 support K8s 1.24?

Reply:
We can backport https://github.com/kubevela/kubevela/pull/4687 and release v1.5.5.  In other words, K8s v1.24 support is added and verified in PR https://github.com/kubevela/kubevela/pull/4687/